src/main/resources/inspectionDescriptions/NamedSymbols.html

```diff
@@ -0,0 +1,11 @@
+<html>
+<body>
+<p>Detects usages of named symbols in wrong contexts and suggests removing the name. <br/>
+    Named symbols can only appear in an alternative, and only if there are no other selected unnamed symbols.
+</p>
```

Comment:
For some reason, there is an extra line at the beginning in the inspections settings description box for each of these inspections. Initially I thought it was due to a newline in the HTML, but this one doesn't seem to have that newline, so maybe it's the `<p>`.